### PR TITLE
Fix/publishing date

### PR DESCRIPTION
**Pull request by armandfardeau**
#### :tophat: What? Why?
Related to : #38 
image
image

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -626,12 +660,15 @@ fr:
         shared:
           info_question:
             created_at: Date de création
+            first_interacted_at: First interacted at
```
Comment (moustachu) [inline config/locales/fr.yml]: 🛑 Please directly add French translation ... Adding missing keys with non-translated values will provoke more issues related to translations from client ...

File: config/locales/fr.yml [hunk 2]
```diff
@@ -626,12 +660,15 @@ fr:
...
             info_question_bottom:
               created_at: Date de soumission
+              first_interacted_at: First interacted at
```
Comment (moustachu) [inline config/locales/fr.yml]: 🛑 Please directly add French translation ... Adding missing keys with non-translated values will provoke more issues related to translations from client ...

File: config/locales/fr.yml [hunk 3]
```diff
@@ -626,12 +660,15 @@ fr:
...
               published_at: Date de publication
               question_votes_count: Décompte des votes pour la question
             info_question_bottom_full:
               created_at: Date de soumission
+              first_interacted_at: First interacted at
```
Comment (moustachu) [inline config/locales/fr.yml]: 🛑 Please directly add French translation ... Adding missing keys with non-translated values will provoke more issues related to translations from client ...

File: config/locales/fr.yml [hunk 4]
```diff
@@ -817,6 +858,7 @@ fr:
             comments: Commentaires
             created_at: Soumise le
             endorsements: Soutiens
+            first_interacted_at: First interacted at
```
Comment (moustachu) [inline config/locales/fr.yml]: 🛑 Please directly add French translation ... Adding missing keys with non-translated values will provoke more issues related to translations from client ...

**Review by moustachu — CHANGES_REQUESTED** [review]
So ... I do understand that the solution here is : 
- to add a new date for Questions : `first_interacted_at`
- replace the value displayed for the "Publication Date" with `first_interacted_at`
- remove the custom behavior that was updating the "Publication date" on state changes 

If I did understand correctly then it's OK ✅ ( ⚠️ After the fixes on FR i18n keys ⚠️ )

File: config/locales/fr.yml (hunk 1 above)
Comment (armandfardeau) [reply config/locales/fr.yml]: Done

**Review by moustachu — APPROVED** [review]

**Pull request merged**